data/json/items/comestibles.json

```diff
@@ -992,8 +992,10 @@
         "charges" : 7,
         "phase" : "liquid",
         "comestible_type" : "DRINK",
-        "flags" : ["BREW"],
-        "brew_time" : 7200
+        "brewable" : {
+            "time" : 7200,
+            "result" : "whiskey"
```

Comment:
OK

This will require adding few extra steps. I'll move the brewing recipes to a new file.

How to use `jq` to style the JSONs?